consensus/consensus-types/src/vote_data.rs

```diff
@@ -39,7 +39,6 @@ impl VoteData {
         Self { proposed, parent }
     }
 
-    #[cfg(any(test, feature = "fuzzing"))]
```

Comment:
Add it back?

Reply:
I needed to use VoteData::dummy() when constructing WrappedLedgerInfo in a few places. So, I removed this.